docker-compose.override.yml

```diff
@@ -13,20 +13,11 @@ services:
         "
     depends_on:
       - storage-emulator
-    env_file:
-      - .env
 
   storage-emulator:
     image: mcr.microsoft.com/azure-storage/azurite
     volumes:
-      - "blobStorage:/data"
+      - "./blobStorage:/data"
     ports:
-      - "10000:10000"  # Blobs
-    command: [
-        "azurite",
-        "--blobHost", "0.0.0.0",
-        "--oauth", "basic",
```

Comment:
`--oauth basic` may be necessary for azurite to work properly. Did you test it without, and confirm that it works as expected?

Reply:
Tested ok.
Following this doc; https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite#oauth-configuration
I see no reason why we should enable the flag.

Reply:
Ok, it seemed to be necessary, when I was figuring out how to set it up.